src/data/redux.test.jsx

```diff
@@ -736,4 +736,6 @@ describe('Data layer integration tests', () => {
       expect(beforeState).not.toEqual(afterState); // Test that the state was updated when polled
     });
   });
+  // TODO: Insert tests somewhere in here that just test `generateAccessibilityString()` to make sure it works right.
+  // TODO: Also, move that function to the api.js? maybe??? idk.
```

Comment:
I think it would be worth it to add some additional tests here for generateAccessibilityString, especially to increase code coverage